### PR TITLE
Refactored porting_anisotropic_segmentation.markdown file

### PR DESCRIPTION
**Pull request by iamrajiv**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by iamrajiv** [comment]
@alalek Please review.

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown [hunk 1]
```diff
@@ -41,8 +41,8 @@ With G-API, we can define it as follows:
 
 It is important to understand that the new G-API based version of
 calcGST() will just produce a compute graph, in contrast to its
-original version, which actually calculates the values. This is a
-principal difference -- G-API based functions like this are used to
+the original version, which actually calculates the values. This is a
+the principal difference -- G-API based functions like this are used to
```
Comment (alalek) [inline doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown:45]: > its the
> a the

Doesn't look valid, IMHO

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown [hunk 2]
```diff
@@ -41,8 +41,8 @@ With G-API, we can define it as follows:
 
 It is important to understand that the new G-API based version of
 calcGST() will just produce a compute graph, in contrast to its
-original version, which actually calculates the values. This is a
-principal difference -- G-API based functions like this are used to
+its original version, which actually calculates the values. This is a
+a principal difference -- G-API based functions like this are used to
```
Comment (alalek) [inline doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown:45]: > its its
> a a

this change looks wrong

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown [hunk 3]
```diff
@@ -336,22 +336,22 @@ terminate called after throwing an instance of 'std::logic_error'
 Aborted (core dumped)
 ```
 
-Fluid backend has a number of limitations in OpenCV 4.0 (see this
+The fluid backend has a number of limitations in OpenCV 4.0 (see this
```
Comment (alalek) [inline doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown:339]: "Fluid" is a name of G-API backend here

**Comment by JoeHowse** [comment]
On the whole, the editing in this pull request is poor.  It adds mistakes of its own and it leaves some other mistakes unfixed.  The resulting text would need to be re-edited by someone else.  I suggest closing the pull request as invalid.

**Pull request closed without merging**